dll/cpl/sysdm/environment.c

```diff
@@ -1827,6 +1852,10 @@ EnvironmentDlgProc(HWND hwndDlg,
 {
     PENVIRONMENT_DIALOG_DATA DlgData;
     DlgData = (PENVIRONMENT_DIALOG_DATA)GetWindowLongPtr(hwndDlg, DWLP_USER);
+    WCHAR szBuffer[MAX_STR_LENGTH];
+    WCHAR szUserName[MAX_STR_LENGTH];
+    WCHAR szFormat[MAX_STR_LENGTH];
+    DWORD Length = MAX_STR_LENGTH - sizeof(WCHAR);
```

Comment:
`dw` is probably the most useless Hungarian prefix in existence. It should be either `cb` or `cch` -- which will actually make you think about what unit this variable's value is measured in. And if you pay attention you will find a bug: `MAX_STR_LENGTH` is a cch, `sizeof(WCHAR)` is a cb, so this calculation makes no sense.

Reply:
> `dw` is probably the most useless Hungarian prefix in existence. It should be either `cb` or `cch` -- which will actually make you think about what unit this variable's value is measured in. And if you pay attention you will find a bug: `MAX_STR_LENGTH` is a cch, `sizeof(WCHAR)` is a cb, so this calculation makes no sense.

I neved used Hungarian notation before. I was just trying to keep the code in it's own standard.
For me Hungarian notation was just to keep track of variables kind. Guess it can also be used for their intention then.

About that cb/cch diff. Should ``MAX_STR_LENGTH - 1`` be used instead ?